.github/workflows/build.yml

```diff
@@ -29,14 +29,14 @@ env:
 
 jobs:
   macos-jdk8:
```

Comment:
We may also rename the job name. As Butao mentioned, should we consider maintaining two JDK versions in parallel? I am not sure whether it is realistic or not.

Reply:
Good catch! I think support and old JDK's are too costly. Currently the community hardly can maintain many aspect of the code that is should be.

Reply:
Thanks @okumin for catching this, updated the job name